code/modules/SCP/SCPs/SCP-2398.dm

```diff
@@ -13,29 +13,19 @@
 	SCP = /datum/scp/scp_2398
 
 /obj/item/weapon/twohanded/scp_2398/attack(mob/living/target,	mob/living/user)
-	var/mob/living/carbon/human/H = target
-	var/mob/living/carbon/human/G = user
+	var/mob/living/carbon/human/T = target
+	var/mob/living/carbon/human/U = user
 	var/activehand = BP_L_ARM //determine which hand to burn (stolen from 113 code because bay code is messy)
-	if(!G.hand)
+	if(!U.hand)
 		activehand = BP_R_ARM
-	if(ishuman(user && target))
-		to_chat(H, SPAN_USERDANGER("Someone begins swinging a bat at you!"))
-		G.visible_message(SPAN_DANGER("[G] begins to swing [src] at [H]!"))
-		if(do_after(G, 4 SECONDS, H))
-			var/obj/item/organ/external/E = G.get_organ(activehand)
-			E.fracture()
-			explosion(H, 1, 1, 3, 3, 1)
-			admin_attack_log(G, H, null, null, "[G] has attacked [H] with SCP-2398!")
-			message_staff("[G] (ckey: [G.ckey]) has swung SCP-2398 at [H] ([H.ckey])!") //"im sure theres a proc to format this for admins already; go find it" no.
-	else
-		if(iscarbon(G))
-			to_chat(H, SPAN_USERDANGER("Someone begins swinging a bat at you!"))
-			G.visible_message(SPAN_DANGER("[G] begins to swing [src] at [H]!"))
-			if(do_after(G, 4 SECONDS, H))
-				var/obj/item/organ/external/E = G.get_organ(activehand)
-				E.fracture()
-				explosion(H, 1, 1, 3, 3, 1)
-				admin_attack_log(G, H, null, null, "[G] has attacked [H] with SCP-2398!")
-				message_staff("[G] (ckey: [G.ckey]) has swung SCP-2398 at [H] ([H.ckey])!")
-				qdel(H) //perhaps a bad solution but i dont want to work with it
+	if(isSCP(target) || !ishuman(user))
 		return
+	to_chat(T, SPAN_USERDANGER("Someone begins swinging a bat at you!"))
+	U.visible_message(SPAN_DANGER("[U] begins to swing [src] at [T]!"))
+	if(do_after(U, 4 SECONDS, T))
+		var/obj/item/organ/external/E = U.get_organ(activehand)
+		E.fracture()
+		explosion(T, 1, 1, 3, 3, 1)
+		admin_attack_log(U, T, null, null, "[U] has attacked [T] with SCP-2398!")
+		message_staff("[U] (ckey: [U.ckey]) has swung SCP-2398 at [T] ([T.ckey])!") //"im sure theres a proc to format this for admins already; go find it" no.
```

Comment:
""im sure theres a proc to format this for admins already; go find it" no." yes

Reply:
Ok actually no, lol, we need to port this shit, fine

Reply:
lmao, this was actually what seven wrote. I told him to try to find a proc to format it but @MACIEKBAKI still passed it. Who knew we never even had the proc.